packer/install-dependencies.sh

```diff
@@ -41,6 +41,88 @@ for KUBERNETES_VERSION in ${K8S_VERSIONS}; do
     pullHyperkube
 done
 
+DASHBOARD_VERSIONS="1.8.3 1.6.3"
+for DASHBOARD_VERSION in ${DASHBOARD_VERSIONS}
+    pullImg "k8s.gcr.io/kubernetes-dashboard-amd64:v${DASHBOARD_VERSION}"
```

Comment:
where is the "not golden image" equivalent for this?

Reply:
There is none. This is additional foo for VHD images: when the k8s cluster comes online its component images will already be local. For non-VHD use-cases the functionality will be as it is today, where the initial bootstrapping process will include fetching docker images.